src/components/Layout/HomeContent.js

```diff
@@ -131,100 +131,104 @@ export function HomeContent() {
+              label="Belajar React">
+              Belajar React
             </ButtonLink>
             <ButtonLink
               href={'/reference/react'}
               type="secondary"
               size="lg"
               className="w-full sm:w-auto justify-center"
-              label="API Reference">
-              API Reference
+              label="Referensi API">
+              Referensi API
             </ButtonLink>
           </div>
         </div>
 
         <Section background="left-card">
           <Center>
-            <Header>Create user interfaces from components</Header>
+            <Header>Membuat antarmuka pengguna dari komponen</Header>
             <Para>
-              React lets you build user interfaces out of individual pieces
-              called components. Create your own React components like{' '}
-              <Code>Thumbnail</Code>, <Code>LikeButton</Code>, and{' '}
-              <Code>Video</Code>. Then combine them into entire screens, pages,
-              and apps.
+              React memungkinkan Anda membangun antarmuka penguna dari
+              bagian-bagian yang disebut komponen. Buat komponen React Anda
+              sendiri seperti <Code>Thumbnail</Code>, <Code>LikeButton</Code>,
+              dan <Code>Video</Code>. Kemudian gabungkan komponen-komponen
+              tersebut ke dalam seluruh layar, halaman, dan aplikasi.
             </Para>
           </Center>
           <FullBleed>
             <Example1 />
           </FullBleed>
           <Center>
             <Para>
-              Whether you work on your own or with thousands of other
-              developers, using React feels the same. It is designed to let you
-              seamlessly combine components written by independent people,
-              teams, and organizations.
+              Baik Anda bekerja sendiri atau dengan ribuan pengembang lain,
+              menggunakan React akan terasa sama. React dirancang untuk
+              memungkinkan Anda menggabungkan komponen yang dirancang oleh
+              pengembang independen, tim, dan organisasi besar secara mulus.
             </Para>
           </Center>
         </Section>
 
         <Section background="right-card">
           <Center>
-            <Header>Write components with code and markup</Header>
+            <Header>Menulis komponen dengan kode dan markup</Header>
             <Para>
-              React components are JavaScript functions. Want to show some
-              content conditionally? Use an <Code>if</Code> statement.
-              Displaying a list? Try array <Code>map()</Code>. Learning React is
-              learning programming.
+              Komponen React adalah fungsi JavaScript. Ingin menampilkan
+              beberapa konten secara kondisional? Gunakan pernyataan{' '}
+              <Code>if</Code>. Menampilkan sebuah daftar? Gunakan fungsi{' '}
+              <Code>map()</Code> ke sebuah senarai. Belajar React adalah belajar
```

Comment:
@resir014 To introduce uncommon word.

```suggestion
              <Code>map()</Code> ke sebuah senarai (<i>array</i>). Belajar React adalah belajar
```